auth2client/src/components/Signin.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+class Signin extends Component {
+    state = {
+        username: "",
+        password: ""
+    };
+    
+
+    render() {
+        return (
+        <form onSubmit={this.handleSubmit}>
+
+            <div className="form-username">
+            <label htmlFor="username">Username</label>
+            <input
+                name="username"
+                value={this.state.username}
+                onChange={this.handleChange}
+                type="text"
+            />
+            </div>
+
+            <div>
+            <label htmlFor="password">Password</label>
+            <input
+                name="password"
+                value={this.state.password}
+                onChange={this.handleChange}
+                type="text"
+            />
+            </div>
+
+            <div>
+            <button type='submit'>Sign In</button>
+            </div>
+
+        </form>
+        );
+    }
+
+    handleChange = e => {
+        e.preventDefault();
+        this.setState({ [e.target.name] : e.target.value })
+    }
+
+    handleSubmit = e => {
+        e.preventDefault();
+        const creds = this.state;
+        const endpoint = 'http://localhost:4200/api/login';
+        axios.post(endpoint, creds)
+        .then(res => {
+            console.log('response data: ', res.data);
```

Comment:
When done testing make sure to get rid of all unnecessary console.log statements before committing code to Github. Another way to take care of console.log statements is in a function to have a debug variable that is set to true. Then wrap your console.log statements in an if(debug){console.log("Hello")} and then when finished testing set debug back to false